tests/fixtures/analog/uplinkCommands.js

```diff
@@ -0,0 +1,102 @@
+export const uplinkSimpleCommands = {
+    'new event': {
+        name: 'newEvent',
+        hex: {
+            dump: '1506081e000973cbe1',
+            command: '15 06 08 1e 00 09 73 cb',
+            lrc: '0xe1'
+        },
+        base64: 'FQYIHgAJc8vh',
+        expectedParameters: {
```

Comment:
```suggestion
        parameters: {
```
?

Other test suite can have "parameters" as source and hex as "expected" values.

Reply:
renamed